### ISSUE TITLE
[BUG] Edit Employee Information

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#4802]
Linda E. has questions regarding entering 3 new employees in the HR Employee page

I entered three employees:
1)	Jennifer Golech
2)	Jennifer Newell
3)	Jeffrey Stensland

Here are several questions: 
1)	My name showed up strictly for Ms. Golech and not the other two.  Should I be concerned? **No your name will appeared in the Modified By field only after you Edit and Save Employee Information.**
2)	On what screen am I allowed to enter Job Title for all three employees? **Need to verify how Job Title is populated. Job Title is not a field that can be populated on the New Employee or Edit Employee page.**
3)	I don’t have an assigned PIDM for Ms. Golech at this time.  Will I be allowed to add it several weeks from now? **Need to determine if Employee ID can be added to the Edit Employee Information page**
4)	Lastly, the Ben White Office is showing up at Jennifer Newell’s Office in CTM’s email data.  If that a drop down field you will be adding. **Need to determine if there are ATD employees at the Ben White Office location**

**Comment by moorerst** [comment]
@SurbhiBakshi I confirmed the ATD employee Jennifer N is working primarily from the Ben White Office.
As requested by Linda E can you add Ben White Office to the Facility drop down list on the New Employee and Edit Employee page?

**Comment by SurbhiBakshi** [comment]
Hi @moorerst - done with making the changes, take a look and let me know if there is anything else.

**Comment by moorerst** [comment]
Hi @SurbhiBakshi the changes look great. I have informed Linda that the changes are in effect. 
Thanks. This can be closed.

**Comment by moorerst** [comment]
@SurbhiBakshi sorry she has requested an additional field to the Edit Employee page.
can you please add Email Address to the Edit Employee page. A lot of this information is not available when a employee is first hired so they need to enter it as they receive it. Thanks

**Comment by SurbhiBakshi** [comment]
@moorerst - no worries! added the Email Address to the Edit Employee form.

**Comment by moorerst** [comment]
@SurbhiBakshi Thanks you can close this issue. If anything else comes up I'll open a new issue
